[2024-03-17T02:27:16.308+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reddit_to_s3.brdev manual__2024-03-17T02:27:04.790231+00:00 [queued]>
[2024-03-17T02:27:16.318+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reddit_to_s3.brdev manual__2024-03-17T02:27:04.790231+00:00 [queued]>
[2024-03-17T02:27:16.319+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-17T02:27:16.329+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): brdev> on 2024-03-17 02:27:04.790231+00:00
[2024-03-17T02:27:16.335+0000] {standard_task_runner.py:60} INFO - Started process 284 to run task
[2024-03-17T02:27:16.347+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'reddit_to_s3', 'brdev', 'manual__2024-03-17T02:27:04.790231+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/ingest.py', '--cfg-path', '/tmp/tmpqtc2zpkc']
[2024-03-17T02:27:16.351+0000] {standard_task_runner.py:88} INFO - Job 64: Subtask brdev
[2024-03-17T02:27:16.387+0000] {task_command.py:423} INFO - Running <TaskInstance: reddit_to_s3.brdev manual__2024-03-17T02:27:04.790231+00:00 [running]> on host ee31986955d4
[2024-03-17T02:27:16.447+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='reddit_to_s3' AIRFLOW_CTX_TASK_ID='brdev' AIRFLOW_CTX_EXECUTION_DATE='2024-03-17T02:27:04.790231+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-17T02:27:04.790231+00:00'
[2024-03-17T02:27:23.807+0000] {reddit_api.py:51} INFO - brdev_2024-03-17.csv already exists in S3
[2024-03-17T02:27:23.809+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-17T02:27:23.816+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=reddit_to_s3, task_id=brdev, execution_date=20240317T022704, start_date=20240317T022716, end_date=20240317T022723
[2024-03-17T02:27:23.852+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-17T02:27:23.874+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
